src/main/java/com/gregtechceu/gtceu/data/recipe/misc/VanillaStandardRecipes.java

```diff
@@ -462,6 +486,16 @@ private static void woodRecipes(Consumer<FinishedRecipe> provider) {
                 'W', ItemTags.WOOL, 'S', new ItemStack(Items.STICK), 'B', Creosote.getBucket());
         VanillaRecipeHelper.addShapedRecipe(provider, "soul_torch", new ItemStack(Blocks.SOUL_TORCH, 1), "WB",
                 'W', ItemTags.SOUL_FIRE_BASE_BLOCKS, 'B', new ItemStack(Blocks.TORCH));
+        if (!ConfigHolder.INSTANCE.recipes.hardMiscRecipes) {
```

Comment:
do these still have recipes when hard recipes are enabled?

Reply:
Yes, these are just easy conversion recipes. In HM, you just make the lantern directly using the base recipe but with a soul torch instead